kv/test_raftstore/test_test.go

```diff
@@ -236,10 +239,14 @@ func GenericTest(t *testing.T, part string, nclients int, unreliable bool, crash
 			}
 		})
 
-		if partitions {
+		// enable cases:
+		//               partitions    no partitions
+		//    unreliable       1                1
+		// no unreliable       1                1
```

Comment:
what's the meaning of 1?

Reply:
It is a support matrix, 1 stands for support. It is used to show supported casees.

Reply:
don't think it provides any useful information, let's remove it.